cmd/upgrade/roles/cmd.go

```diff
@@ -919,6 +923,14 @@ func buildOperatorRoleCommandsFromCluster(
 		)
 		_, err = awsClient.IsPolicyExists(policyARN)
 		hasPolicy := err == nil
+		policyAttached := false
+		if hasPolicy && operatorRoleName != "" {
+			for _, policy := range rolePolicyDetails[operatorRoleName] {
+				if policy.PolicyArn == policyARN {
+					policyAttached = true
```

Comment:
Can policyAttached by renamed to something like
DefaultRolePolicyAttached?
The difference between HasPolicy and PolicyAttached isn't clear for me. 😢